dash/App/Chains/Chain/Components/index.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react'
+
+import link from '../../../../../resources/link'
+import svg from '../../../../../resources/svg'
+import RingIcon from '../../../../../resources/Components/RingIcon'
+
+const networkDefaults = {
+  type: 'ethereum',
+  id: 'Chain ID',
+  name: 'Chain Name',
+  explorer: 'Block Explorer',
+  symbol: 'Native Symbol',
+  isTestnet: false,
+  color: 'accent3'
+}
+
+export const SubmitChainButton = ({ text, enabled, textColor, onClick }) => {
+  return (
+    <div
+      role='button'
+      className={enabled ? 'addTokenSubmit addTokenSubmitEnabled' : 'addTokenSubmit'} 
+      style={{ color: textColor }}
+      onClick={(e) => {
+        // Left Click
+        if (e.button === 0) onClick()
```

Comment:
is this necessary when using `onClick`? might only have been needed for `onMouseDown`